test/libsolidity/AnalysisFramework.cpp

```diff
@@ -46,11 +46,13 @@ AnalysisFramework::parseAnalyseAndReturnError(
 	bool _reportWarnings,
 	bool _insertVersionPragma,
 	bool _allowMultipleErrors,
-	bool _allowRecoveryErrors
+	bool _allowRecoveryErrors,
+	bool _insertLicensePragma
```

Comment:
Is there a situation where we would insert the version but not the license pragma?